packages/SwingSet/src/kernel/vatManager/deliver.js

```diff
@@ -44,11 +53,25 @@ export function makeDeliver(tools, dispatch) {
     // TODO: accumulate used.allocate and used.compute into vatStats
   }
 
-  async function doProcess(dispatchRecord, errmsg) {
-    const dispatchOp = dispatchRecord[0];
-    const dispatchArgs = dispatchRecord.slice(1);
-    transcriptManager.startDispatch(dispatchRecord);
-    await runAndWait(() => dispatch[dispatchOp](...dispatchArgs), errmsg);
+  // vatDeliveryObject is one of:
```

Comment:
Did you mean to keep this comment here? It duplicates a lot of info that's now in types; move the prose to the type declaration? Also: cite the `.md` doc?

Reply:
I disagree.  Keep the prose here.  It's more informative and more legible than the type declaration.